README.md

```diff
@@ -706,142 +688,64 @@ try {
 }
 ```
 
-#### Sign example: using promises
-
-```js
-const options = {
-  postPrepareSigningUrl:  "/document/123/sign/prepare",
-  postFinalizeSigningUrl: "/document/123/sign/finalize",
-};
-
-webeid.sign(options)
-.then((status) => { /* HANDLE SUCCESS */ })
-.catch((error) => { /* HANDLE FAILURE */ });
-```
-
-#### Sign example: success
-
-The result of a status check is the status object which contains SemVer strings for the library, browser extension and native application.
+#### Sign example: result
 
-```js
+```json
 {
-  // The body content is useful when signing in a SPA applications.
-  // The server can also send a 204 status with an empty response.
-  body: {
-    documentId: "123"
-  },
-
-  headers: {
-    "content-length": "20",
-    "content-type": "application/json; charset=utf-8",
-    "date": "Thu, 10 Sep 2020 18:28:12 GMT",
-    "etag": "W/\"14-jy7dMclQhztcZaNKkjCj7ZOnAXw\"",
-    "x-firefox-spdy": "h2",
-    "x-powered-by": "Express"
-  },
-
-  ok:         true,
-  redirected: false,
-  status:     200,
-  statusText: "OK",
-  type:       "basic",
-
-  // The server may chooses to redirect the request on the
-  // postFinalizeSigningUrl address. In case of one or more redirects
-  // the URL will be set to the last URL which finally gave an OK status.
-  url: "https://example.com/document/123/sign/finalize"
+  "signature":"W1r4zf+7LkLs8Dy40nR+UOwZ7pkagywNSUxzB5/RJHdeYUZrmnAjGILdt+ek5vfHLBz9bF7B3UCRZWqZS61uNp2xJvjl7SuxpuAeo0lZrwZlvpuC2lT8mcvz9Eqso391",
+  "signatureAlgorithm":{
+    "cryptoAlgorithm":"ECC",
+    "hashFunction":"SHA-256",
+    "paddingScheme":"NONE"
+  }
 }
 ```
 
-#### Sign example: failure
-
-When the signing request fails, in addition to the usual `name`, `message` and `stack` properties, the error object contains additional info **when possible**.  
-See [Known errors](#known-errors) for error `code` options.
+### Migration from API version 1 to version 2
```

Comment:
Maybe it would be better if the migration section was at the start of API section or it's own section before or after the API section. It logically makes sense to have it here, but when a developer scrolls between the API function definitions and Known errors section, having it here means more scrolling.

...is documentation UX even a thing? Ignore me if I'm just being crazy.